src/test/java/com/wikia/webdriver/common/core/helpers/ForgottenPasswordUserPool.java

```diff
@@ -0,0 +1,19 @@
+package com.wikia.webdriver.common.core.helpers;
+
+
+import com.wikia.webdriver.common.core.configuration.Configuration;
+import com.wikia.webdriver.common.properties.Credentials;
+
+public class ForgottenPasswordUserPool {
+
+  private static final Credentials CREDENTIALS = Configuration.getCredentials();
+
+  public static UserWithEmail user1 = new UserWithEmail(User.FORGOTTEN_PASSWORD,
+    CREDENTIALS.forgottenPasswordEmail1Address,
+    CREDENTIALS.forgottenPasswordEmail1Password);
+
+  public static UserWithEmail user2 = new UserWithEmail(User.FORGOTTEN_PASSWORD_SPACES,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make user2 a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this "public static user2" field final [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1444)